frame/election-provider-multi-phase/Cargo.toml

```diff
@@ -40,7 +40,7 @@ hex-literal = "0.3.1"
 substrate-test-utils = { version = "4.0.0-dev", path = "../../test-utils" }
 sp-core = { version = "4.0.0-dev", default-features = false, path = "../../primitives/core" }
 sp-io = { version = "4.0.0-dev", path = "../../primitives/io" }
-sp-npos-elections = { version = "4.0.0-dev", default-features = false, features = [ "mocks" ], path = "../../primitives/npos-elections" }
```

Comment:
i am assuming removing this feature was intentional

Reply:
yes, it is not used anymore.